test/action_test.dart

```diff
@@ -161,7 +161,7 @@ void main() {
         expect(listened, isTrue);
 
         listened = false;
-        action.clearListeners();
+        await action.dispose();
         await action();
         expect(listened, isFalse);
```

Comment:
clearlisteners() is deprecated and it says to instead call dispose(). This is just a warning cleanup